### DISCUSSION TITLE
Help on installing the Latest Hyas and Doks Theme

### DISCUSSION THREAD

**Discussion by technosprout** [gethyas/hyas#430] [Q&A]
Hi There
Can anybody help me with the npm commands to install HYAS
and the [@hyas/doks-core]

The Doks that hyas@latest does not install archetypes or all the layouts. 
is it just a matter of copying the files over?

**Comment by h-enk** [comment]
Yes from `node_modules/@hyas/doks-core`, but copy only what you'd like to customize (so that future updates of `@hyas/doks-core` have their effect)

So, e.g. copy `node_modules/@hyas/doks-core/archetypes/default.md` to `archetypes/default.md`, if you'd like to change that.
